kfs-core/src/main/resources/edu/arizona/kfs/coa/spring-coa.xml

```diff
@@ -21,6 +21,10 @@
                 <value>edu/arizona/kfs/coa/ojb-coa.xml</value>
             </list>
         </property>
+        <property name="triggerNames">

```

Comment:
There aren't actually any triggers defined in spring-coa, so this is an unneeded override.